.github/workflows/workflow.yml

```diff
@@ -50,12 +50,12 @@ jobs:
         os: [ubuntu-24.04, macos-14, windows-2025]
         node-version: ['22']
         # Must include the minimum deno version from the `DENO_VERSION_RANGE` constant in `node/bridge.ts`.
-        deno-version: ['v1.39.0', 'v2.2.4']
+        deno-version: ['v2.4.2']
```

Comment:
This is unfortunate, as it's the only part of this PR that affects users not targeted by the feature flag. But `deno bundle` isn't available in previous versions.